hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestLs.java

```diff
@@ -84,6 +84,7 @@ public void processOptionsNone() throws IOException {
     assertFalse(ls.isOrderSize());
     assertFalse(ls.isOrderTime());
     assertFalse(ls.isUseAtime());
+    assertFalse(ls.isDisplayECPolicy());
```

Comment:
This is the EC policy patch again. I don't see why abfs needs it at all.

Reply:
This was because I involved changes in hadoop-common between trunk and branch-2 due the reason above. I have redone it and it should be excluded now.